docs/modules/reference/pages/billing/signup/subscription.adoc

```diff
@@ -0,0 +1,130 @@
+To change your plan, click btn:[Change Subscription].
+
+
+
+
+== Invoicing Details
+
+Next, fill in your invoicing details:
+
+.Invoicing details
+image::billing/signup/cloud-trial-image3.png[alt="Invoicing details form"]
+
+Contact Name and Email::
+Enter the contact details for receiving notifications and invoices.
+It defaults to your account's email; modify if needed for different invoicing.
+
+Company Name, Address, City, Zip Code, Country::
+Provide your company's invoicing information for the invoice.
+
+Tax ID Type, Tax ID::
+Optionally, if VAT-registered, enter your VAT ID.
+Our payment processor will validate it according to the given ID Type.
+Tax type updates based on the selected country.
+
+Click btn:[Continue to payment] after filling in the details.
+
+
+== Payment method
+
+Finally, authorize a payment method for automated invoice settlement.
+Clicking btn:[Confirm payment details] might redirect you to your bank for payment authorization.
+
+Your card is charged at the trial's end and then at each billing period's start.
+
+.Payment method dialog
+image::billing/signup/cloud-trial-image4.png[]
+
+== Confirmation
+
+Agree to Payara Cloud's Terms and Conditions.
+You can also choose to opt in for product updates via email to complete the process.
+
+.Confirmation
+image::billing/signup/cloud-trial-image5.png[]
+
+== Provisioning
+
+You'll then be directed to the Payara Cloud Application Management console.
+Account provisioning may take a few minutes.
+Welcome to Payara Cloud!
+
+
+== Manage Subscriptions
+
+To see and manage your existing subscriptions, click on the Billing Management menu in the left vertical menu bar.
+Through the Billing Management menu, you can xref:docs:reference:billing/subscription/user-subscription.adoc[manage users] in your subscription, view a list of your active subscriptions and xref:docs:reference:billing/signup/additional.adoc[create new subscriptions].
+
+
+.Manage Subscription
+image::billing/signup/billing-plan-2.png[alt="Payara Cloud Plans]
+
+.Subscription Management Menu
+[checklist]
+* ❶ Click to Manage Users.
+* ❷ Click to create a new subscription
+* ❸ Click to go back to application management
+
+
+=== Edit Subscription Name
+
+You can edit the name of a subscription by clicking on it.
+
+
+
+=== View Current Usage
+You can view your current resource usage for a given subscription by clicking on the subscription name from the subscription list.
+
+
+=== Set Usage Alerts
+To have better visibility and control over your usage of Payara Cloud, you can set usage alerts for your subscriptions.
+
+To create a usage alert, click on the btn:[Create New Alert] button on a given subscription.
+
+.New Usage Alert Button
+image::billing/signup/billing-plan-3.png[alt="New Usage Alert Button]
+
+This will open the new alert form.
+
+.New Alert Form
+image::billing/signup/billing-plan-4.png[alt="New Usage Alert Form]
```

Comment:
Click to open image in new tab change is required as suggested in earlier comment